packages/tools/client-debugger/client-debugger-view/src/components/FluidHandleView.tsx

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+	IDebuggerMessage,
+	InboundHandlers,
+	handleIncomingMessage,
+	HasContainerId,
+	FluidObjectNode,
+	DataVisualizationMessage,
+} from "@fluid-tools/client-debugger";
+import React from "react";
+import { useMessageRelay } from "../MessageRelayContext";
+import { Waiting } from "./Waiting";
+import { waitingLabels } from "./WaitingLabels";
+import { FluidDataView } from "./FluidDataView";
+
+const loggingContext = "EXTENSION(HandleView)";
+
+/**
+ * {@link FluidHandleView} input props.
+ */
+export interface FluidHandleViewProps extends HasContainerId {
+	fluidObjectId: string;
```

Comment:
The debugger package exports a `HasFluidObjectId` interface you can extend here.